packages/actor-query-operation-extend/lib/ActorQueryOperationExtend.ts

```diff
@@ -73,3 +75,7 @@ export class ActorQueryOperationExtend extends ActorQueryOperationTypedMediated<
     };
   }
 }
+
+interface IActorQueryOperationExtendArgs extends IActorQueryOperationTypedMediatedArgs {
+  expressionEvaluatorFactory: ExpressionEvaluatorFactory;
```

Comment:
Just realized that the READMEs of this actor (and probably also other actors) will need to modified to mention this as config param.